jb/itch5/ut_type_based_order_book.hpp

```diff
@@ -0,0 +1,321 @@
+#ifndef jb_itch5_ut_type_based_order_book_hpp
+#define jb_itch5_ut_type_based_order_book_hpp
+
+#include <jb/itch5/map_based_order_book.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+namespace jb {
+namespace itch5 {
+namespace testing {
```

Comment:
Please move this file to jb/itch5/testing/, in general the namespace should match the path of the files.